src/Redux/Crypto.js

```diff
@@ -0,0 +1,27 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = [];
+const URL = 'https://api.coincap.io/v2/assets';
+
+export const getCryptos = createAsyncThunk('getting Api from cryptos', async () => {
+  const gottenCryptos = await axios.get(URL).catch((error) => error);
+  const cryptoData = gottenCryptos.data;
+  return cryptoData.data;
+});
+
+export const searchCryptos = createAsyncThunk('search the cryptos', async (id) => {
+  const exploreCryptos = await axios.get(`https://api.coincap.io/v2/assets/${id}`).catch((error) => error);
```

Comment:
[OPTIONAL]

- I can see that you are passing an `id` parameter to your searchCrypto function, but I cant see anywhere the` id` was passed as the argument or used throughout your app, If the` Id` is not used, kindly remove it so as to prevent any form of confusion in the future 👍🏽